app/src/index.ts

```diff
@@ -20,11 +20,13 @@ let welcomeWindow: BrowserWindow | null = null
 
 declare const MAIN_WINDOW_WEBPACK_ENTRY: string
 
+let homePath = process.env.OLLAMA_HOME || app.getPath('home') + '/.ollama'
```

Comment:
It's not easy to modify this env var (user would need to dig through some launchd stuff), so I'd leave it as-is for the mac app. Instead, we can make it user modifiable through the app itself